Ships as Raw

```diff
@@ -0,0 +1,860 @@
+RearArcConjunction|45
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Quasar Fire II
+Type|Quasar
+Keywords|0
+Faction|Imperial
+Size|Medium
+Points|61
+Hull|6
+defenseTokens|Rd Br
+Command|2
+Squad|4
+Eng|2
+Speed1|2
+Speed2|10
+Speed3|011
+Speed4|0
+FrontShields|2
+SideShields|2
+RearShields|1
+FrontArmament|RRB
+SideArmament|BR
+RearArmament|B
+AntiSquadron|R
+Upgrade|Title Officers Offensive Weapon Team Weapon Team
+FrontArcOffset|35.3
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|9.9
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|56
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|46
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Imperial Star Destroyer Kuat Refit
+Type|Imperial Star Destoryer
+Keywords|0
+Faction|Imperial
+Size|Large
+Points|112
+Hull|11
+defenseTokens|Rd Br Br Cn
+Command|3
+Squad|2
+Eng|4
+Speed1|1
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|4
+SideShields|3
+RearShields|2
+FrontArmament|RRRBBKKK
+SideArmament|RBKK
+RearArmament|RBK
+AntiSquadron|KB
+Upgrade|Title Officers Defensive Offensive Support Team Weapons Team  Ordance Ion Cannon
+FrontArcOffset|41
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|15
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|79
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|49
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Imperial Star Destroyer Cymoon 1 Refit
+Type|Imperial Star Destoryer
+Keywords|0
+Faction|Imperial
+Size|Large
+Points|112
+Hull|11
+defenseTokens|Rd Br Br Cn
+Command|3
+Squad|3
+Eng|4
+Speed1|1
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|4
+SideShields|3
+RearShields|2
+FrontArmament|RRRRRBB
+SideArmament|RBBB
+RearArmament|RBB
+AntiSquadron|KK
+Upgrade|Title Officers Offensive Weapons Team Turbo Laser Turbo Laser Fleet Command
+FrontArcOffset|41
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|15
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|79
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|49
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Imperial I Class Star Destroyer
+Type|Imperial Star Destoryer
+Keywords|0
+Faction|Imperial
+Size|Large
+Points|110
+Hull|11
+defenseTokens|Rd Br Br Cn
+Command|3
+Squad|4
+Eng|4
+Speed1|1
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|4
+SideShields|3
+RearShields|2
+FrontArmament|RRRBBKKK
+SideArmament|RRKK
+RearArmament|RBB
+AntiSquadron|BK
+Upgrade|Title Officers Defensive Offensive Offensive Weapons Team Turbo Laser Defensive Ion Cannon

```

Comment:
IonCannon, WeaponsTeam, Turbolaser, etc.  change to be consistant with whatever you choose.  At the end of the day each card should get one token, or string of chars between spaces.